source/simulator/solver.cc

```diff
@@ -714,8 +714,8 @@ namespace aspect
     SolverControl solver_control_cheap (parameters.n_cheap_stokes_solver_steps,
                                         solver_tolerance);
 
-    SolverControl solver_control_expensive (system_matrix.block(block_vel,block_p).m() +
-                                            system_matrix.block(block_p,block_vel).m(), solver_tolerance);
+    SolverControl solver_control_expensive (parameters.n_expensive_stokes_solver_steps,
```

Comment:
We talked about this briefly with @tjhei @gassmoeller and @jdannberg and came up with this number, but I don't know if it's good. Is setting it to sys...m() + sys...m() as default an option or does the default have to be a fixed number? If the system doesn't converge the error message is "This error message can indicate that you have simply not allowed a sufficiently large number of iterations for your iterative solver to converge." so if someone runs into this issue they might realize that they have to adjust the maximum number of stokes steps. I could also mention this limitation in the parameter description in case people search the manual for bits of this error message.